packages/services/api/src/modules/app-deployments/providers/app-deployments-manager.ts

```diff
@@ -170,11 +238,34 @@ export class AppDeploymentsManager {
       },
     });
 
-    return await this.appDeployments.retireAppDeployment({
+    const result = await this.appDeployments.retireAppDeployment({
       organizationId: target.orgId,
       targetId: target.id,
       appDeployment: args.appDeployment,
     });
+
+    const currentUser = await this.session.getViewer();
```

Comment:
create an app deployment is done via a target access token, so there is no viewer and this will reject.

Reply:
TODO @TuvalSimha: remove audit log creation for app deployments